book/visual-effects.md

```diff
@@ -1785,7 +1784,7 @@ text and boxes but also:
 - user-configurable blending modes via `mix-blend-mode`;
 - rounded rectangle clipping via destination-in blending or direct clipping;
 - optimizations to avoid surfaces when possible;
-- and browser compositing with extra surfaces for faster scrolling.
+- browser compositing with extra surfaces for faster scrolling.
```

Comment:
I think we leave the and in a lot of prior chapters. We should standardize. Could be another PR